presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -41,23 +45,34 @@
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return PrestoSparkQueryRunner.createHivePrestoSparkQueryRunner();
+        Map<String, String> configs = new HashMap<>();
+        // prevent to use the default Prestissimo config files since the Presto-Spark will generate the configs on-the-fly.
+        configs.put("catalog.config-dir", "/");
+        return PrestoSparkQueryRunner.createHivePrestoSparkQueryRunner(configs);
     }
 
     @Test
-    public void testNativeExecutionWithTableWrite()
+    public void testNativeExecutionWithProjection()
     {
+        assertUpdate("CREATE TABLE test_order WITH (format = 'DWRF') as SELECT orderkey, custkey FROM orders LIMIT 100", 100);
+
+        String prestoServerPath = System.getProperty("PRESTO_SERVER");
+        assertNotNull(prestoServerPath);
```

Comment:
nit: Add a message with general asserts to make the error messages more meaningful.